examples/time-of-day/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -0,0 +1,131 @@
+  - name: time-of-day-check-template
+    metadata:
+      annotations:
+        argo-hub-template/description: 'Checks the day and time and fails if the current time is outside of a specified window or day of the week'
+        argo-hub-template/icon_url: 'https://cdn.jsdelivr.net/gh/codefresh-io/argo-hub@main/examples/time-of-day-check/assets/icon.svg'
+        argo-hub/icon_background: "#f4f4f4"
+    inputs:
+      parameters:
+      - name: early
+        value: "{{workflow.parameters.TIME_WINDOW_STARTS}}"
+      - name: late
+        value: "{{workflow.parameters.TIME_WINDOW_FINISHES}}"
+      - name: inverted
+        value: "{{workflow.parameters.TIME_WINDOW_INVERTED}}"
+      - name: days
+        value: "{{workflow.parameters.TIME_WINDOW_DAYS}}"
+      - name: timezone
+        value: "{{workflow.parameters.TIME_WINDOW_TIMEZONE}}"
+    script:
+      image: debian:latest
+      command: [bash]
+      source: |
+        echo "Checking time window:"
+        echo "  Selected time zone is {{inputs.parameters.timezone}}"
+        echo "  Time window starts at {{inputs.parameters.early}} and finishes at {{inputs.parameters.late}}"
+        echo "    (format should be 24 hour time, HH:MM)"
+        echo "  Valid days are {{inputs.parameters.days}}"
+        echo ""
+        if [ '{{inputs.parameters.inverted}}' = 'false' ]; then 
+          echo "The window is not inverted, so if we are inside the window we can proceed"
+        else 
+          echo "The window is inverted. We need to be outside the window for the pipeline to proceed (inverted != false)"
+        fi
+        echo "Current container time is $(date)"
+        echo "Time in {{inputs.parameters.timezone}} is $(TZ={{inputs.parameters.timezone}} date)"
+        DAY=$(TZ={{inputs.parameters.timezone}} date +%A)
+        NOW=$(TZ={{inputs.parameters.timezone}} date +%s)
+        EARLY=$(TZ={{inputs.parameters.timezone}} date +%s -d "{{inputs.parameters.early}}")
+        LATE=$(TZ={{inputs.parameters.timezone}} date +%s -d "{{inputs.parameters.late}}")
+        if [[ "{{inputs.parameters.days}}" == *"$DAY"* ]];
+        then
+          echo "$DAY's are included"
+        else
+          echo ""
+          echo "Above settings do not include $DAY. Exiting."
+          exit 1
+        fi
+        INSIDE_TIME_WINDOW='false'
+        if (( $EARLY < $NOW && $NOW < $LATE )); then
+          INSIDE_TIME_WINDOW='true'
+        fi
+        echo "Inside time window? $INSIDE_TIME_WINDOW"
+        echo ""
+        if [ $INSIDE_TIME_WINDOW = '{{inputs.parameters.inverted}}' ]; then
+          echo "Current time is outside of the time window defined above"
+          exit 2
+        else
+          echo "Pipeline is allowed to proceed"
+        fi
+        
+  - name: whalesay-template
+    inputs:
+      parameters:
+      - name: message
+    container:
+      image: docker/whalesay
+      command: [cowsay]
+      args: ["{{inputs.parameters.message}}"]
+      
+  - name: example-workflow
```

Comment:
Need an entry in the README.md and a new example-workflow.md file in the docs folder

Reply:
What level of detail do I need to put into the example-workflow.md file? Does it get rendered into a page somewhere, or are a couple of lines sufficient?

Reply:
not a lot, it just need to be there

Reply:
I added the 2 missing files as placeholder